nessie-iceberg/settings.gradle.kts

```diff
@@ -16,13 +16,12 @@
 
 import java.util.Properties
 
+includeBuild("../build-logic") { name = "nessie-build-logic" }
+
 if (!JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_11)) {
   throw GradleException("Build requires Java 11")
 }
 
-// Needed by NesQuEIT's manageNessieProjectDependency() in its settings.gradle.kts
```

Comment:
Appreciated side-effect of the move to `build-logic` is that this hack can disappear.